packages/core-components/package.json

```diff
@@ -3,7 +3,7 @@
   "description": "Reusable Tailor UI components",
   "author": "ExtensionEngine <info@extensionengine.com>",
   "license": "MIT",
-  "version": "1.0.0-beta.6",
+  "version": "1.0.0-beta.7",
```

Comment:
There are [versions](https://www.npmjs.com/package/@tailor-cms/core-components) published after this one, please bump it to .11 😉

Reply:
Addressed in 6a8070e